spec/features/students_can_pair_spec.rb

```diff
@@ -12,10 +12,23 @@
     find(:css, "##{partner.id}").click
 
     expect(page).to have_content('Complete')
+    expect(page).to_not have_content('Paired?')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.